app/2.0/toolbox/prpl.md

```diff
@@ -68,32 +68,45 @@ When you generate an App Toolbox project using Polymer CLI, the new project cont
 
 The shell is responsible for routing and usually includes the main navigation UI for the app.
 
-The app should call `importHref` to lazy-load fragments as they're required. For example, when the
-user changes to a new route, it imports the fragment(s) associated with that route. This may
-initiate a new request to the server, or simply load the resource from the cache.
+The app should lazy load code at runtime as required to get a large performance
```

Comment:
"The app should lazy load code at runtime" --> "The app should lazy-load code at runtime as required to get a large performance" or maybe "The app should lazily load code at runtime as required to get a large performance"